### PR TITLE
OpeningHour i18n: until -> reopening at

### PR DESCRIPTION
**Pull request by G-Ray**

### CODE REVIEW

File: src/components/OpeningHour.jsx [hunk 1]
```diff
@@ -39,7 +39,9 @@ const OpeningHour = ({ schedule, showNextOpenOnly = false }) => {
     {label}
     {(nextTransition &&
      (!showNextOpenOnly || showNextOpenOnly && status === 'closed')) &&
-    ` - ${_('until {nextTransitionTime}', 'hour panel', { nextTransitionTime: nextTransition })}`
+    ` - ${_('reopening at {nextTransitionTime}', 'hour panel', {
+      nextTransitionTime: nextTransition,
+    })}`
```
Comment (amatissart) [inline src/components/OpeningHour.jsx:44]: If status is `open`, the message would be wrong:
"Open - reopening at 20:00" , instead of "Open - until 20:00"

Comment (G-Ray) [reply src/components/OpeningHour.jsx]: Thank you @amatissart , I updated the pull request adding an element `NextTransition` to keep things more pleasant to read

File: src/components/OpeningHour.jsx [hunk 2]
```diff
@@ -35,12 +35,24 @@ const OpeningHour = ({ schedule, showNextOpenOnly = false }) => {
     </div>;
   }
 
+  const NextTransition = () => {
+    if (!nextTransition || showNextOpenOnly && status !== 'closed') {
+      return null;
+    }
+
+    const text = status === 'close'
```
Comment (amatissart) [inline src/components/OpeningHour.jsx:43]: `close` :arrow_right: `closed`

File: src/components/OpeningHour.jsx [hunk 3]
```diff
@@ -35,12 +35,24 @@ const OpeningHour = ({ schedule, showNextOpenOnly = false }) => {
...
+      ? 'reopening at {nextTransitionTime}'
+      : 'until {nextTransitionTime}';
+
+    return ` - ${_(text,
```
Comment (amatissart) [inline src/components/OpeningHour.jsx:47]: Using a variable here will prevent `xgettext` from extracting the translation keys automatically.

Comment (G-Ray) [reply src/components/OpeningHour.jsx]: True, I updated

File: src/components/OpeningHour.jsx [hunk 4]
```diff
@@ -35,12 +35,23 @@ const OpeningHour = ({ schedule, showNextOpenOnly = false }) => {
...
+  const NextTransition = () => {
+    if (!nextTransition || showNextOpenOnly && status !== 'closed') {
+      return null;
+    }
+
+    const options = { nextTransitionTime: nextTransition };
+
+    const text = status === 'closed'
+      ? ` - ${_('reopening at {nextTransitionTime}', 'hour panel', options)}`
+      : ` - ${_('until at {nextTransitionTime}', 'hour panel', options)}`;
```
Comment (amatissart) [inline src/components/OpeningHour.jsx:47]: typo: `until at` :arrow_right: `until`

And then we should be ok :smiley:

Comment (G-Ray) [reply src/components/OpeningHour.jsx:47]: my bad, I should take a deep breath before re-reading...

**Pull request merged**